dev/doc/universes.md

```diff
@@ -22,7 +22,7 @@ Universes
 
 Universe instances (an array of levels) gets substituted when
 unfolding definitions, are used to typecheck and are unified according
-to the rules in the ITP'14 paper on universe polymorphism in Coq.
+to the rules in the ITP'14 paper on universe polymorphism in Rocq.
```

Comment:
This one is dubious.

Reply:
Good catch, a precise citation would probably be have been better, but Coq is probably closer from the actual paper title than Rocq anyway.